gensrc/proto/lake_service.proto

```diff
@@ -20,6 +20,8 @@ message PublishVersionRequest {
 
 message PublishVersionResponse {
     repeated int64 failed_tablets = 1;
+    // Mapping from tablet id to compaction score.
+    map<int64, double> compaction_scores = 2;
```

Comment:
Should add modifier required/optional

Reply:
> Should add modifier required/optional

Maps cannot be `repeated`, `optional` or `required`. 
https://developers.google.com/protocol-buffers/docs/proto#maps